ref/general/mp-openapi.adoc

```diff
@@ -10,47 +10,80 @@
-
-MP OpenAPI processes JAX-RS annotations by default. You can augment the existing code with OpenAPI annotations, which provide detailed documentation for the REST API.
-
-Vendors must process the relevant JAX-RS annotations like @Path, @Consumes, and Java objects used as input or output to JAX-RS operations. Developers can deploy the existing JAX-RS application into an MP OpenAPI vendor and check out the output from `/openapi`.
-
-Annotations like @Callback, @Components, @ParameterIn, and more are found in the Eclipse OpenAPI annotations package. For an overview of annotation, see https://download.eclipse.org/microprofile/microprofile-open-api-1.0/microprofile-openapi-spec.html#_quick_overview_of_annotations.
-
-When the same annotation is used on a class and a method, the values from the method instance take precedence for that particular method, for example, @Server and @Tag annotations. The annotation values from the method’s parameters take precedence over corresponding annotation values from that method, for example, @Parameter and @RequestBody. The @Schema annotation has a complex set of possible combinations, and can be placed on POJOs and referenced from several other annotations.
-
-For examples of annotations, see link:https://github.com/eclipse/microprofile-open-api/wiki/Annotation-Samples[Annotation Samples]
-
-== Static OpenAPI files
-
-Take the initial output from `/openapi` as a starting point to document APIs with Static OpenAPI files. You can write the static files before any code. Enterprises often adopt this approach to lock-in the API contract. The client and provider must abide by this OpenAPI document, referred to as the "source of truth".
+OpenAPI, previously known as Swagger, defines a standard interface for documenting and exposing RESTful APIs. This standard interface enables both humans and computers to process a web service available through its REST API without directly accessing the source code of the web service. MicroProfile OpenAPI provides a set of Java interfaces and programming models that enable Java developers to natively produce OpenAPI documents from their JAX-RS applications.
+
+Developers can communicate with the REST API in their web service only if they have the relevant information to do so. It is important to document APIs when they are published so that other developers can use them.
+
+There are two main ways to generate a link:https://swagger.io/docs/specification/about/[OpenAPI] document. The developer can generate documentation of the REST API from the source code of the REST API itself. The reference document thus produced, lists all the API endpoints and descriptions of how to use them. Alternatively, the developer can start by designing the API in the form of structured documentation that they can then augment with annotations in the code. The developer can even generate the stubs for the source code, including client libraries for the API, from this manually created documentation.
```

Comment:
In the phrase, "The reference document thus produced", I think "thus" could be replaced with a more common word to ensure consistency and clarity. Maybe change to "The reference document is then produced".